website/sidebars.js

```diff
@@ -168,6 +168,46 @@ const sidebarSettings = {
             "docs/build/groups",
           ],
         },
+        {
+          type: "category",
```

Comment:
Are there any concerns about moving the semantic layer docs this drastically? Will it impact people's ability to find them? We're taking them from the top level and putting them into a category that may not be the first place the existing clientele will go to look for them.

Reply:
yea good point, its here because this will only be beta available for cli users on 1.6 and higher. the old docs will disappear once we go commercial release i think because we don't want users to define metrics using the old spec. 

i think its also a little diff to the existing SL with the new revamp - what do you think @azzam34 ?